lib/logatron/kv_transformer.rb

```diff
@@ -0,0 +1,21 @@
+module Logatron
+  class KvTransformer
```

Comment:
I would consider making this function static. Since it will create a new KvTransformer for every invocation of the block, it should be more performant since you are not creating a new object every time